### ISSUE TITLE
Random Newline

### ISSUE THREAD

**Issue by MaximilianEmel** [jjcm/nonio#137]
Sometimes a newline gets added to the end of a text content (by the editor, presumably), and isn't truncated server-side, either.

**Comment by jjcm** [comment]
Good call. I probably want to truncate extra newlines/whitespace on submission regardless.
